hack/common

```diff
@@ -188,6 +188,34 @@ function push_image() {
     skopeo copy --dest-tls-verify=false docker-daemon:"$1" docker://"$2"
 }
 
+# debug_print [max_count [interval]]
+function debug_print () {
+  local filename=$(basename "$0")
+  local max=${1:-120}
+  local interval=${2:-10}
+  while [ $max -gt 0 ];
+  do
+    clopod=$( oc -n ${NAMESPACE} get pods -o name | egrep "cluster-logging-operator" ) || :
+	clo=$( echo $clopod | awk -F'/' '{print $2}' )
+    if [ -n "$clo" ]; then
+        date >> $ARTIFACT_DIR/$filename.clo.log || :
+        oc -n ${NAMESPACE} logs $clo >> $ARTIFACT_DIR/$filename.clo.log || :
+        echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.clo.log || :
+        date >> $ARTIFACT_DIR/$filename.clo.images || :
+        oc -n ${NAMESPACE} exec $clo -- env | egrep _IMAGE >> $ARTIFACT_DIR/$filename.clo.images || :
+        echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.clo.images || :
+    fi
+    date >> $ARTIFACT_DIR/$filename.events.log || :
+    oc -n ${NAMESPACE} get events >> $ARTIFACT_DIR/$filename.events.log || :
+    echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.events.log || :
+    date >> $ARTIFACT_DIR/$filename.all.log || :
+    oc -n ${NAMESPACE} get all >> $ARTIFACT_DIR/$filename.all.log || :
+    echo "-------------------------------------------------" >> $ARTIFACT_DIR/$filename.all.log || :
+    sleep $interval
+    max=$( expr $max - 1 )
```

Comment:
```
     max=$( expr $max - 1 ) || :
```
`expr` will return an error to the shell if its result has the value of `0`, so use `|| :` to "catch" and ignore that error